nodejs/eks/cluster.ts

```diff
@@ -447,6 +472,8 @@ export function createCore(name: string, args: ClusterOptions, parent: pulumi.Co
     return {
         vpcId: pulumi.output(vpcId),
         subnetIds: pulumi.output(subnetIds),
+        publicSubnetIds: pulumi.output(publicSubnetIds),
+        privateSubnetIds: pulumi.output(privateSubnetIds),
```

Comment:
Feels like these should be `args.publicSubnetIds ? pulumi.output(args.publicSubnetIds) : undefined` and args.privateSubnetIds ? pulumi.output(args.privateSubnetIds) : undefined`, respectively.

Reply:
Done